chirp-impl/src/test/java/sample/chirper/chirp/impl/ChirpServiceTest.java

```diff
@@ -37,28 +58,28 @@ public static void tearDown() {
     }
 
     @Test
-    public void shouldPublishShirpsToSubscribers() throws Exception {
+    public void shouldPublishChirpsToSubscribers() throws Exception {
         ChirpService chirpService = server.client(ChirpService.class);
         LiveChirpsRequest request = new LiveChirpsRequest(TreePVector.<String>empty().plus("usr1").plus("usr2"));
-        Source<Chirp, ?> chirps1 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(3, SECONDS);
+        Source<Chirp, ?> chirps1 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(1, SECONDS);
         Probe<Chirp> probe1 = chirps1.runWith(TestSink.probe(server.system()), server.materializer());
         probe1.request(10);
-        Source<Chirp, ?> chirps2 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(3, SECONDS);
+        Source<Chirp, ?> chirps2 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(1, SECONDS);
         Probe<Chirp> probe2 = chirps2.runWith(TestSink.probe(server.system()), server.materializer());
         probe2.request(10);
 
         Chirp chirp1 = new Chirp("usr1", "hello 1");
-        chirpService.addChirp("usr1").invoke(chirp1).toCompletableFuture().get(3, SECONDS);
+        chirpService.addChirp("usr1").invoke(chirp1).toCompletableFuture().get(firstRequestOnEntityTimeout, SECONDS);
```

Comment:
I would also expect `secondRequestOnEntityTimeout` here instead of `firstRequestOnEntityTimeout`. 

Actually, I may be missing something? Once we pass the canary setup it should be ok to use a lower timeout, no? Do you observe failures on each first iteration with any given entity?